be/src/formats/orc/apache-orc/c++/src/ColumnReader.cc

```diff
@@ -674,6 +674,10 @@ class StringDictionaryColumnReader : public ColumnReader {
 private:
     std::shared_ptr<StringDictionary> dictionary;
     std::unique_ptr<RleDecoder> rle;
+    std::unique_ptr<RleDecoder> lengthDecoder;
```

Comment:
If these variables are only used by `StringDictionaryColumnReader::loadDictionary`, maybe you can extract them as class like `class DictionaryLazyLoader` for high cohesion.

Reply:
But `DictionaryLazyLoader` is used only inside `ColumReader.cc` and used only by `StringDictionaryColumnReader`, I think for simplicity we don't need to create another class. Maybe in the future, if we have more operations on DictionaryLoader, we can do that.

Reply:
emm, this is for maintainability, not for reuasability

Reply:
OK. I'll fix for that.

Reply:
But again, I feel quite awkawrd and unnecessary. Because dictionary loader and stirngdictionrycolumreader are cohensive, no need to separate them.